crates/bevy_render/src/render_phase/draw.rs

```diff
@@ -42,6 +58,24 @@ pub trait PhaseItem: Send + Sync + 'static {
     fn sort_key(&self) -> Self::SortKey;
     /// Specifies the [`Draw`] function used to render the item.
     fn draw_function(&self) -> DrawFunctionId;
+
+    /// Specifies what kind of sort to apply to the phase. Generally if the same type
+    /// implements [`BatchedPhaseItem`], this should return [`RenderPhaseSortMode::Stable`].
+    /// In almost all other cases, this should not be altered from the default,
+    /// [`RenderPhaseSortMode::Unstable`], as this provides the best balance of CPU and GPU
+    /// performance.
+    ///
+    /// It's generally only advised to use [`RenderPhaseSortMode::Unsorted`] if and only if
+    /// the renderer supports a depth prepass, which is by default not supported by the rest
+    /// of Bevy's first party rendering crates. Even then, this may have a negative impact
+    /// on GPU-side perf due to overdraw.
+    ///
+    /// It's advised to always profile for performance changes when changing this to
+    /// looser values.
```

Comment:
```suggestion
    /// It's advised to always profile for performance changes when changing this to
    /// a different value.
```
I'm not sure what "looser" is supposed to mean here. By default it's unstable, so would bot stable and unordered be looser than that?

Reply:
Perhaps "more unconstrained"? Not sure how to best put it. The fewer constraints on the sort order ,the faster it tends to go, but as mentioned above it's a tradeoff depending on what the setup for the app looks like.

Reply:
If they were ordered by looseness/constraints, I think it would be stable -> unstable -> unordered. As the default is unstable, there are one value more constrained and one less

Reply:
Yes, that's the intention. Swapping to Stable would be a deliberate choice to reduce sort phase performance for some other property elsewhere.

Reply:
yup, so it's not only changing to a "looser" value, it's changing to a different value as you could pick one that is looser (unordered) or one that is less loose (stable)

Reply:
Alright done.